docs/getting-started/javascript-options-api.md

```diff
@@ -156,129 +159,158 @@ Now that you've installed all required packages, you are ready to add the Kendo
+    },
     ```
- 
-These steps let you render a very basic Grid by running `npm run dev` and navigating to the local URL displayed in the terminal.
- 
+
+After completing all the steps above, your `App.vue` will look like this:
+
+```js
+  <script>
+    import "@progress/kendo-theme-default/dist/all.css";
+    import products from "./appdata/products.json";
+    import categories from "./appdata/categories.json";
+    import { process } from "@progress/kendo-data-query";
+    import { Grid } from "@progress/kendo-vue-grid";
+
+    export default {
+      name: "App",
+      components: {
+        grid: Grid,
+      },
+      data: function () {
+        return {
+          categories: categories,
+          products: products,
+          columns: [
+            { field: "ProductName", title: "Product Name" },
+            { field: "UnitPrice", title: "Price" },
+            { field: "UnitsInStock", title: "Units in Stock" },
+            { field: "Discontinued" },
+          ],
+        };
+      },
+    };
+  </script>
+
+  <template>
+    <h1>Hello Kendo UI for Vue!</h1>
+    <grid :data-items="products" :columns="columns"></grid>
+  </template>
+```
+
+This sample code lets you run an application with a very basic Grid:
+
+1. Execute the `npm run dev` command.
+1. Navigate to the local URL displayed in the terminal.
+
 > Notice the `No valid license found` message and the watermark in the Grid. They are informational and encourage you to activate your trial or commercial license and to [add a license file to your application](slug:my_license_vue). Once you complete these licensing steps, the license message and the watermark will disappear.
  
 ## Configure the Vue Data Grid
 
-Now that you have a running Grid, you are ready to use some of its basic features like sorting and paging:
-
-1. In the Grid declaration, add paging, sorting, and a height style that activates scrolling.
-
-    ```html
-    <template>
-      <h1>Hello Kendo UI for Vue!</h1>
-      <grid
-        :data-items="products"
-        :columns="columns"
-        :pageable="pageable"
-        :sortable="sortable"
-        :style="{ height: '400px' }"
-      ></grid>
-    </template>
-    ```
+Now that you have a running Grid, you are ready to use some of its basic features like sorting and paging.
 
 1. Implement the paging and sorting functionality in the `data` option:
-   
-    - Set the page size (`take`) to 10.
-    - Set the initial `skip` for the paging.
-    - Set the initial sorting by Product name.
+
     - Set `sortable` to `true`.
     - Set `pageable` to `true`.
-    - Initialize the `dataResult` empty array.
+    - Set the page size (`take`) to `10` to define how many items are rendered on each page.
+    - Set the initial `skip` value, which allows the Grid to calculate how many items to skip when paging.
+    - Set the initial sorting by product name.
+    - Initialize the `dataResult` empty array, which returns the results from the sorting and paging.
```

Comment:
the `dataResult` doesn't really return the results from sorting and paging, it rather holds them.